app/models/idea.rb

```diff
@@ -17,6 +17,12 @@ def submitted?
     submission_date.present?
   end
 
+  def approved?
+    return false if status == 'awaiting_approval' || status.nil?
```

Comment:
How about we default newly created ideas to `awaiting_approval`? That way we can get rid of this method and call the `#awaiting_approval?` method provided by enums.

It's also worth noting that this method overrides the `#approved?` method created by the enum helpers. Could cause confusion in the future.

Reply:
Renamed method to approved_by_admin? to avoid overriding the enum method.  I'll change the statement to 'awaiting_approval' || 'draft' once that change is complete.